features/support/env.rb

```diff
@@ -139,3 +144,13 @@ def extract_process_id(data)
   stdout = aruba.command_monitor.find(Aruba.platform.detect_ruby(last_command_started)).send(:stdout)
   expect(@service_current_process_id).not_to eq(extract_process_id(stdout))
 end
+
+Then(/^the box is "([^"]*)"$$/) do |box_name|
```

Comment:
Not quite sure what you do here. Seems self fullfilling. you need to set the current box somewhere here - https://github.com/projectatomic/vagrant-service-manager/blob/master/features/support/env.rb#L71 and then check against here.

this should most likely be:

Given /current box is (.*)/ do |current_box|

and skipping if there is no match between the box yu set earlier and the box specified in this clause.